src/ast/irbuilderbpf.cpp

```diff
@@ -1474,6 +1474,24 @@ Value *IRBuilderBPF::CreatKFuncArg(Value *ctx,
   return expr;
 }
 
+Value *IRBuilderBPF::CreateRawTracepointArg(Value *ctx, const std::string &builtin)
+{
+  // argX
+  int offset = atoi(builtin.substr(3).c_str());
+  llvm::Type *type = getInt64Ty();
+
+  ctx = CreatePointerCast(ctx, type->getPointerTo());
+  Value *expr = CreateLoad(type,
+                             CreateGEP(type, ctx, getInt64(offset)),
+                             builtin);
+
+  // LLVM 7.0 <= does not have CreateLoad(*Ty, *Ptr, isVolatile, Name),
+  // so call setVolatile() manually
+  dyn_cast<LoadInst>(expr)->setVolatile(true);
```

Comment:
We don't support LLVM below 10 anymore, so it's fine to drop this in favor of `CreateLoad(*Ty, *Ptr, isVolatile, Name)` call.